wiki/Beatmap_Spotlights/pt-br.md

```diff
@@ -0,0 +1,342 @@
+---
+tags:
+  - Charts
+  - Ranking Charts
+  - Seasonal Spotlights
+---
+
+# Beatmap Spotlights
+
+![Beatmap Spotlights logo](img/logo.jpg)
+
+Os **Beatmap Spotlights** (também conhecidos como *Spotlights* ou *Ranking Charts*) é um projeto de curadoria recorrente para recomendação e destaque de [beatmaps](/wiki/Beatmaps) pela seu design e gameplay excelentes e únicos. É acompanhado por uma [liga de temporada](#spotlights-seasons) que dá recompensas para todos os jogadores participantes.
+
+**A iteração atual desse projeto está em desenvolvimento e muitos de suas funcionalidades ainda estão sendo implementadas.** Específicos desse sistema, tais como recompensas, rankings sazonais ou funcionalidades de jogabilidade, podem mudar a qualquer momento.
+
+A temporada atual é a temporada de Verão 2020. É considerada a ser o primeiro teste do novo sistema de Beatmap Spotlights.
```

Comment:
```suggestion
A temporada atual é a temporada de Verão 2020. É considerada o primeiro teste do novo sistema de Beatmap Spotlights.
```
Ease of comprehension

Reply:
I actually had a lot of trouble trying to figure out something for this one. Thanks!